test/views/html/MainViewSpec.scala

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.html
+
+import config.{ConfigDecorator, LocalTemplateRenderer}
+import controllers.auth.requests.UserRequest
+import models.UserDetails
+import org.jsoup.nodes.{Document, Element}
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import play.twirl.api.Html
+import uk.gov.hmrc.auth.core.retrieve.Credentials
+import uk.gov.hmrc.auth.core.retrieve.v2.TrustedHelper
+import uk.gov.hmrc.play.bootstrap.binders.SafeRedirectUrl
+import uk.gov.hmrc.renderer.TemplateRenderer
+import util.Fixtures.{buildFakeRequestWithVerify, fakeNino}
+import util.UserRequestFixture.buildUserRequest
+
+class MainViewSpec extends ViewSpec {
+
+  override protected def localGuiceApplicationBuilder(): GuiceApplicationBuilder =
+    GuiceApplicationBuilder()
+      .overrides(
+        bind(classOf[TemplateRenderer]).to(classOf[LocalTemplateRenderer])
```

Comment:
LocalTemplateRenderer requires the service to be running, best move these tests to the IT folder